components/RecipeDetails.js

```diff
@@ -44,12 +53,20 @@ const StyledImage = styled(Image)`
 `;
 
 const StyledDescription = styled.p`
-  margin: 2em 2em;
+  margin: 1rem;
 `;
 
 const StyledLink = styled(Link)`
-  border: 1rem solid black;
+  all: unset;
   background-color: #000;
+  padding: 5px 32px;
+  font-size: 1rem;
+  border-radius: 0.4em;
   color: #fff;
-  text-decoration: none;
+`;
```

Comment:
```suggestion
&:hover {
  cursor: pointer;
}
`;
```
You probably want to show the user in any way that the element is clickable - it doesn't have to be a changing cursor of course though. I know that most of your other buttons and links already do this, so maybe it is because you use `all: unset` for this styled component.